source_gen/lib/src/builder.dart

```diff
@@ -139,15 +139,17 @@ class _Builder extends Builder {
     try {
       genPartContent = formatOutput(genPartContent);
     } catch (e, stack) {
-      log.severe(
-          'Error formatting generated source code for ${library.identifier} '
-          'which was output to ${outputId.path}.\n'
-          'This may indicate an issue in the generated code or in the '
-          'formatter.\n'
-          'Please check the generated code and file an issue on source_gen if '
-          'appropriate.',
-          e,
-          stack);
+      log.warning(
```

Comment:
This is also making it a warning and not failure which seems like the larger change, please indicate that in the changelog and commit message.

Reply:
Oops! Not intended. I played w/ that idea a bit, but I don't think we want to turn these into silent failures.